extensions/kind/src/create-cluster.ts

```diff
@@ -174,16 +182,15 @@ export async function createCluster(
       errorMessage = error;
     }
 
-    telemetryLogger.logError('createCluster', {
-      provider,
-      httpHostPort,
-      httpsHostPort,
-      ingressController,
-      error: errorMessage,
-      stdErr: errorMessage,
-    });
+    telemetryOptions.error = errorMessage;
+    telemetryOptions.stdErr = errorMessage;
```

Comment:
not sure why we have two field with the same object ?

Reply:
Solely because 'it was there before' :) and I was trying to keep the PR to the smallest change. Fixed to only add one attribute (and change to error as per next comment).